src/logo/index.js

```diff
@@ -4,50 +4,55 @@ import React from 'react';
 // Internal Dependencies.
 import './style.scss';
 
-const Logo = ( { className, text, src, alt, onClick, href } ) => {
+const Logo = ( { className, text, src, alt, onClick, href, extraText } ) => {
 	if ( ! text && ! src ) {
 		return '';
 	}
 
 	return (
-		<span
-			className={ `stc-logo ${ className ? className : '' }` }
-			onClick={ ( event ) => {
-				if ( 'function' === typeof onClick ) {
-					onClick( event );
-				}
-			} }
-		>
-			{ src ? (
-				<>
-					{ href ? (
-						<a
-							href={ href }
-							target="_blank"
-							rel="noopener noreferrer"
-						>
+		<>
+			<span
+				className={ `stc-logo ${ className ? className : '' }` }
+				onClick={ ( event ) => {
+					if ( 'function' === typeof onClick ) {
+						onClick( event );
+					}
+				} }
+			>
+				{ src ? (
+					<>
+						{ href ? (
+							<a
+								href={ href }
+								target="_blank"
+								rel="noopener noreferrer"
+							>
+								<img
+									src={ src }
+									className="stc-logo-image"
+									alt={ alt }
+								/>
+							</a>
+						) : (
 							<img
 								src={ src }
 								className="stc-logo-image"
 								alt={ alt }
 							/>
-						</a>
-					) : (
-						<img
-							src={ src }
-							className="stc-logo-image"
-							alt={ alt }
-						/>
-					) }
-				</>
-			) : null }
+						) }
+					</>
+				) : null }
 
-			{ text ? (
-				<div className="stc-logo-text" alt={ alt }>
-					{ text }
-				</div>
-			) : null }
-		</span>
+				{ text ? (
+					<div className="stc-logo-text" alt={ alt }>
+						{ text }
+					</div>
+				) : null }
+			</span>
+			{ extraText && (
+				<span className="stc-extra-text">{ extraText } </span>
```

Comment:
Can we add this text using psuedo class?

Reply:
I think the pseudo-class is not useful here because we are displaying that extra text based on props. If the extra text props are passed from the ST plugin, then it will be displayed in the ST component.

Reply:
In ST component you can simple add the class which will add a psedu element using CSS instead of HTML.

Reply:
Awesome!